### PR TITLE
[PT-6422][add] push/pull source of newly added UI Definition object

### PR DESCRIPTION
**Pull request by deborovik**

### CODE REVIEW

File: src/types/ui.types.ts [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
 export type UiMetadata = Omit<UiDefinition, 'children'> & {
   children: string[];
 };
 
 export interface UiDefinition {
   name: string;
   version: number;
+  default?: boolean;
+  primary?: boolean;
+  referenceId: string;
```
Comment (smankevics) [inline src/types/ui.types.ts:22]: Could you please add another type just to not mix UIDefinition with SF entity:

```
export interface UIDefinitionContainerDTO {
  id: string;
  modelId: string;
  sourceBlob: string;
  version?: number;
  referenceId: string;
}
```

UiDefinition itself shouldn't be changed.

File: src/types/ui.types.ts [hunk 2]
```diff
@@ -1,12 +1,25 @@
...
+}
+
 export type UiMetadata = Omit<UiDefinition, 'children'> & {
   children: string[];
 };
 
 export interface UiDefinition {
   name: string;
   version: number;
+  default?: boolean;
```
Comment (smankevics) [inline src/types/ui.types.ts:20]: Keep just `primary`. No need to have `default`

File: src/common/pull.ui.ts [hunk 3]
```diff
@@ -26,26 +38,45 @@ export async function pullUI(params: CommandParams): Promise<string[]> {
...
+  }[] = await Promise.all(
     uiDefProductModels.map((productModel) =>
-      fetchDocumentContent(conn, productModel.VELOCPQ__UiDefinitionsId__c).then((content) => ({
-        content,
-        productModel,
-      })),
+      fetchUiDefinitions(conn, productModel.Id, productModel.VELOCPQ__Version__c)
+        .then((uiDefinitions) => {
+          return uiDefinitions;
+        })
```
Comment (smankevics) [inline src/common/pull.ui.ts:49]: Is this needed?

Comment (deborovik) [reply src/common/pull.ui.ts]: Deleted

File: src/types/ui.types.ts (hunk 1 above)
Comment (deborovik) [reply src/types/ui.types.ts]: Done

File: src/types/ui.types.ts (hunk 2 above)
Comment (deborovik) [reply src/types/ui.types.ts]: Done

**Review by smankevics — APPROVED** [review]

**Pull request merged**